src/test/scenarios/managed-network/output/src/managed-network/azext_managed_network/generated/custom.py

```diff
@@ -65,6 +72,9 @@ def managed_network_scope_assignments_create(cmd, client,
                                              scope_assignment_name,
                                              location=None,
                                              parameters_assigned_managed_network=None):
+    parameters = {}
+    parameters['location'] = location  # string
+    parameters['assigned_managed_network'] = parameters_assigned_managed_network  # string

```

Comment:
this part really don't need for now. the python sdk operation definition is the same as custom.py operation definition. both of them will be flattened or unflattened the same time consistently.
If you want to save them for subcommand etc. then you need flatten them in az first.

Reply:
yes, I will remove that